Tests/Tests/Utils/CDTChangedDictionaryJSONWrappingTests.m

```diff
@@ -0,0 +1,88 @@
+//
+//  CDTChangedDictionaryJSONWrappingTests.m
+//  Tests
+//
+//  Created by Michael Rhodes on 17/08/2015.
+//  Copyright (c) 2015 Cloudant. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+//  except in compliance with the License. You may obtain a copy of the License at
+//    http://www.apache.org/licenses/LICENSE-2.0
+//  Unless required by applicable law or agreed to in writing, software distributed under the
+//  License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+//  either express or implied. See the License for the specific language governing permissions
+//  and limitations under the License.
+//
+
+#import <XCTest/XCTest.h>
+
+#import <CDTChangedDictionary.h>
+#import <CDTChangedArray.h>
+
+@interface CDTChangedDictionaryJSONWrappingTests : XCTestCase
+
+@property (nonatomic, strong) CDTChangedDictionary *dictionary;
+
+@end
+
+@implementation CDTChangedDictionaryJSONWrappingTests
```

Comment:
would it be useful to test the `changed` true → false transition which occurs when calling `dictionaryWrappingContents`?